src/components/editor/PropertyTemplateOutline.jsx

```diff
@@ -93,115 +98,136 @@ export class PropertyTemplateOutline extends Component {
     }
   }
 
-  handleComponentCase = (property) => {
+  /**
+   * returns jsx to render for passed property
+   * @private
+   */
+  propertyComponentJsx = (property) => {
     let rtReduxPath = Object.assign([], this.props.reduxPath)
```

Comment:
This could be changed into a `const`; the `rtReduxPath` is an array that only gets added to in the function